services/gateway/apis/http-version3/swagger/definitions/transactions.json

```diff
@@ -590,6 +598,11 @@
 						"type": "string",
 						"example": "0"
 					},
+					"fee": {
```

Comment:
We had removed fee since it was not used. But incase we are adding it back to swagger then should maybe we add to gateway as well?
https://github.com/LiskHQ/lisk-service/blob/release/0.7.0/services/gateway/apis/http-version3/methods/transactionsEstimateFees.js#L31

Reply:
Done!